src/test/java/Pages/GooglePage.java

```diff
@@ -0,0 +1,71 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GooglePage {
+
+    private static final By REJECT_ALL = By.id("W0wltc");
+
+
+    private static final By SEARCH_FIELD = By.id("APjFqb");
+    private static final By SEARCH_RESULTS = By.xpath("//a/h3");
+    private static final By FEELING_LUCKY = By.id("gbqfbb");
+    //redirected to url that contains search term
+    private WebDriver driver;
+
+    public GooglePage() {
+        this.driver = new ChromeDriver();
+    }
+
+    public void navigateToGoogle() {
+        driver.manage().window().maximize();
+        driver.get("https://www.google.co.uk/");
+    }
+
+
+    public void selectRejectAll() {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
```

Comment:
Nicely wrapped up wait here, however, we could make this reusable and use it on other elements as well. Maybe think about creating a `waitForElementToBeClickable(By locator()` to parametrise this method

Reply:
Then try using that in other methods in this class